src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/codeSmell/ComparisonOperandsOrderInspector.java

```diff
@@ -79,60 +83,24 @@ public void visitPhpBinaryExpression(BinaryExpression expression) {
                     return;
                 }
 
-                if (PREFER_YODA_STYLE && isRightConstant) {
+                if (optionPreferYodaStyle && isRightConstant) {
                     holder.registerProblem(expression, messageUseYoda, ProblemHighlightType.WEAK_WARNING, new TheLocalFix());
                     return;
                 }
-                if (PREFER_REGULAR_STYLE && isLeftConstant) {
+                if (optionPreferRegularStyle && isLeftConstant) {
                     holder.registerProblem(expression, messageUseRegular, ProblemHighlightType.WEAK_WARNING, new TheLocalFix());
                 }
             }
         };
     }
 
     public JComponent createOptionsPanel() {
-        return (new ComparisonOperandsOrderInspector.OptionsPanel()).getComponent();
-    }
-
-    private class OptionsPanel {
-        final private JPanel optionsPanel;
-
-        final private JCheckBox preferYodaStyle;
-        final private JCheckBox preferRegularStyle;
-
-        public OptionsPanel() {
-            optionsPanel = new JPanel();
-            optionsPanel.setLayout(new MigLayout());
-
-            preferYodaStyle    = new JCheckBox("Prefer yoda style", PREFER_YODA_STYLE);
-            preferRegularStyle = new JCheckBox("Prefer regular style", PREFER_REGULAR_STYLE);
-
-            preferYodaStyle.addChangeListener(e -> {
-                PREFER_YODA_STYLE = preferYodaStyle.isSelected();
-                if (PREFER_YODA_STYLE) {
-                    preferRegularStyle.setSelected(false);
-                    PREFER_REGULAR_STYLE = false;
-                }
-
-                CONFIGURED = PREFER_YODA_STYLE || PREFER_REGULAR_STYLE;
-            });
-            optionsPanel.add(preferYodaStyle, "wrap");
-
-            preferRegularStyle.addChangeListener(e -> {
-                PREFER_REGULAR_STYLE = preferRegularStyle.isSelected();
-                if (PREFER_REGULAR_STYLE) {
-                    preferYodaStyle.setSelected(false);
-                    PREFER_YODA_STYLE = false;
-                }
-
-                CONFIGURED = PREFER_YODA_STYLE || PREFER_REGULAR_STYLE;
+        return OptionsComponent.create((component) -> {
+            component.createRadio((radioComponent) -> {
```

Comment:
Once opted for this, we should consider that it will have _default checked option_ when this inspection is enabled. I don't think that is need deactivate this inspection by default. Then... could we consider the use the regular style here? The yoda style is more relevant on C language. (Except if you really prefer disable it, but for a big surprise, I don't know about this feature until I work on it, and I was really willing to open an issue to implement it)